src/pages/Location/LocationConfirmation/LocationConfirmation.js

```diff
@@ -95,16 +116,15 @@ const LocationConfirmation = () => {
                   >
                     Cooperative Extension Service office
                   </a>
-                  , or
```

Comment:
again here, it now says `Disclaimer: Consult your local [NRCS Service Center](https://www.nrcs.usda.gov/wps/portal/nrcs/detailfull/national/programs/financial/csp/?&cid=nrcsdev11_000242), [Cooperative Extension Service office](https://nifa.usda.gov/land-grant-colleges-and-universities-partner-website-directory), or [Conservation District](https://www.nacdnet.org/general-resources/conservation-district-directory/) for detailed guidance.`